### PR TITLE
Added Praveen's details.

### PR DESCRIPTION
**Pull request by praveenscience**
Added just the three links.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/329"></a>

### CODE REVIEW

File: public/data/praveenscience.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://twitter.com/praveenscience",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn more about Web Development on my YouTube channel",
+      "url": "https://rb.gy/fupw21",
+      "icon": "youtube"
+    }
+  ],
+  "milestones": []
```
Comment (aayush89890) [inline public/data/praveenscience.json:27]: Hi @praveenscience , Welcome to the community.
Can you please remove this line? There is no need for this.
```suggestion
```

File: public/data/praveenscience.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Praveen Kumar Purushothaman",
+  "bio": "Full Stack JavaScript Specialist",
+  "avatar": "https://i.imgur.com/wbUfjzdl.png",
```
Comment (aayush89890) [inline public/data/praveenscience.json:4]: It would be great if you can replace 
```suggestion
  "avatar": "https://github.com/praveenscience.png",
```

**Review by aayush89890 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

File: public/data/praveenscience.json (hunk 2 above)
Comment (praveenscience) [reply public/data/praveenscience.json]: But why? I would like the other pic. The pic I have got here (https://i.imgur.com/wbUfjzdl.png) will be replacing the GitHub ones soon.

**Comment by praveenscience** [comment]
@aayush89890 @Panquesito7 Is there an issue in using Imgur pic? 🤔

**Comment by CodingSpecies** [comment]
> 
> 
> @aayush89890 @Panquesito7 Is there an issue in using Imgur pic? 🤔

@praveenscience I think this is because if you update your GitHub profile picture, it will update on LinkFree as well. Thank you.

**Comment by praveenscience** [comment]
@CodingSpecies updated that. Hope this is now accepted? 😊

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**